scripts/build-crossdock.sh

```diff
@@ -21,10 +21,7 @@ else
   exit 0
 fi
 
-docker build -f crossdock/Dockerfile \
-	--build-arg tornado=$TORNADO \
-	--tag $REPO:$COMMIT .
-
+# This image is already built in `make crossdock` step
+docker tag crossdock_python:latest $REPO:$COMMIT
 docker tag $REPO:$COMMIT $REPO:$TAG
-docker tag $REPO:$COMMIT $REPO:gh-$GITHUB_RUN_NUMBER
 docker push $REPO
```

Comment:
Q: how would docker know that we're pushing `crossdock_python:latest`? Seems like a name mismatch.

Reply:
Docker will not push crossdock_python. It will only push all the tags for `$REPO` image i.e. `jaegertracing/xdock-py:<commit>` and `jaegertracing/xdock-py:<tag>`